src/components/BuyListElement/BuyListElement.tsx

```diff
@@ -0,0 +1,22 @@
+import { CloseOutlined } from "@ant-design/icons";
+import styles from "./BuyListElement.module.css";
+import { useStore } from "../../store/store";
+
+interface Props {
```

Comment:
Component Interface names should be name as "Component name + Props"

Example: **BuyListElementProps**  

You also have to be consistent throughout the codebase in the naming of the Props